src/jit/codegenlegacy.cpp

```diff
@@ -19201,12 +19201,14 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                 {
                     noway_assert(helperNum != CORINFO_HELP_UNDEF);
 
+                    #ifdef FEATURE_READYTORUN_COMPILER
```

Comment:
Nit: I think that the convention is to start the ifdefs at the start of the line